operator/v1/0000_90_cluster_csi_driver_01_config.crd.yaml

```diff
@@ -57,6 +57,14 @@ spec:
           spec:
             description: spec holds user settable values for configuration
             properties:
+              disableStorageClass:
+                default: false
+                description: DisableStorageClass determines if creation of storageclass
```

Comment:
I think the correct resource name is `StorageClass`

Reply:
I am happy to change but they seem to be using interchangeably in k8s and other places where we document stuff. For description purposes, it should not matter?